provd/debian/rules

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@ --without=autoreconf,makefile
+
```

Comment:
In general, you should vendor the dependency in Go to avoid packaging them all. See the recipe in some other projects like adsys or pro for windows.